manifests/init.pp

```diff
@@ -438,6 +440,7 @@
   String[1] $service_user                                        = 'redis',
   Optional[Integer[0]] $service_timeout_start                    = undef,
   Optional[Integer[0]] $service_timeout_stop                     = undef,
+  Optional[Integer] $service_oom_score_adjust                    = undef,
```

Comment:
are negative and positive integers allowed? Otherwise we could enforce this via the datatype.